fossunited/fossunited/doctype/foss_event_grant/foss_event_grant.json

```diff
@@ -149,8 +154,9 @@
   {
    "fieldname": "grant_amount",
    "fieldtype": "Select",
+   "in_standard_filter": 1,
    "label": "Grant amount",
-   "options": "\nRs. 10,000\nRs. 25,000\nRs. 50,000"
+   "options": "\nRs. 10,000\nRs. 25,000\nRs. 50,000\nCustom"
   },
```

Comment:
Why don't we have just a custom amount field rather than having a select, then custom. We can just put the amount directly. It's going to be used internally anyway